client/me/notification-settings/blogs-settings/style.scss

```diff
@@ -123,3 +123,8 @@
 		height: 22px;
 	}
 }
+
+.notification-settings__back-button {
```

Comment:
I would suggest modifying the selector like so. This way we can avoid the usage of  the infamous`!important`. 🙂
```
.notification-settings__back-button.components-button.has-icon.has-text {
  padding: 0;
  margin-bottom: 24px;
}
```

Reply:
Indeed, I'll change that.

Reply:
Fixed!